apis/v1alpha1/generator.yaml

```diff
@@ -223,6 +237,8 @@ resources:
           service_name: acm
           resource: Certificate
           path: Status.ACKResourceMetadata.ARN
+      DistributionConfig.Origins.Items.CustomOriginConfig.HTTPSPort:
+        go_tag: json:"httpSPort,omitempty"
```

Comment:
it's sad that we have to do this but, good catch!

Reply:
sometimes i wish we had a bot to catch these, it's just very hard to keep up with all the fields.